src/utils/utils.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export namespace Utils {
```

Comment:
Is it a general utils or only for Azure?
If it is a general utils, then merge with `generalUtils.ts`.
If it is only for Azure, better rename it.

Reply:
merged into generalUtils.ts